internal/ignition/ignition.go

```diff
@@ -1435,6 +1441,22 @@ func SetHostnameForNodeIgnition(ignition []byte, host *models.Host) ([]byte, err
 	return configBytes, nil
 }
 
+func (ib *ignitionBuilder) shouldAppendOKDFiles(ctx context.Context, infraEnv *common.InfraEnv, cfg IgnitionConfig) bool {
+	// OKD_RPMS_IMAGE explicitly set in config
+	if cfg.OKDRPMsImage != "" {
+		return true
+	}
+	// Selected payload contains `okd-rpms` image
+	releaseImage, err := ib.versionHandler.GetReleaseImage(infraEnv.OpenshiftVersion, infraEnv.CPUArchitecture)
+	if err != nil {
```

Comment:
Does this case relevant to a "happy-flow" use-case, or is it a real indication of an unexpected error?

Reply:
Its unexpected error. If we can't inspect the release image for some reason we should not embed OKD Ignition bits and complicate troubleshooting (also, unittests are making use of it)

Reply:
maybe we should log it or propagate the error?